src/server_manager/web_app/digitalocean_app/create_server_app.ts

```diff
@@ -0,0 +1,75 @@
+/*
+  Copyright 2018 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+import {customElement, html, LitElement, property} from 'lit-element';
+
+import {ManagedServerRepository} from '../../model/server';
+import {OutlineNotificationManager} from '../ui_components/outline-notification-manager';
+import {Location, OutlineRegionPicker} from '../ui_components/outline-region-picker-step';
+
+// DigitalOcean mapping of regions to flags
+const FLAG_IMAGE_DIR = 'images/flags';
+const DIGITALOCEAN_FLAG_MAPPING: {[cityId: string]: string} = {
+  ams: `${FLAG_IMAGE_DIR}/netherlands.png`,
+  sgp: `${FLAG_IMAGE_DIR}/singapore.png`,
+  blr: `${FLAG_IMAGE_DIR}/india.png`,
+  fra: `${FLAG_IMAGE_DIR}/germany.png`,
+  lon: `${FLAG_IMAGE_DIR}/uk.png`,
+  sfo: `${FLAG_IMAGE_DIR}/us.png`,
+  tor: `${FLAG_IMAGE_DIR}/canada.png`,
+  nyc: `${FLAG_IMAGE_DIR}/us.png`,
+};
+
+@customElement('digital-ocean-create-server')
+export class DigitalOceanCreateServer extends LitElement {
+  @property({type: Function}) localize: Function;
+  @property({type: Object}) notificationManager: OutlineNotificationManager = null;
```

Comment:
You can remove the annotation for this one, since you never set it in HTML.

Reply:
I'm not sure why setting the property isn't working. I suspect that it's because the `OutlineNotificationManager` isn't ready when we do the property binding. Have you seen this behavior before? Do you have any ideas how to make this work with property binding?

If not, we can just add a separate `initialize(...)` method (or setter)?